lib/cli/index.js

```diff
@@ -3847,6 +3848,33 @@ export async function createPythonBom(path, options) {
           const basePath = dirname(f);
           let reqData;
           let frozen = false;
+
+          reqData = readFileSync(f, { encoding: "utf-8" });
+          const directDependencies = await parseReqFile(reqData, true);
+          if (directDependencies?.length) {
+            // Mark direct dependencies from requirements.txt as manifest-analysis
+            directDependencies.forEach((pkg) => {
```

Comment:
Shall we add this evidence creation in the `parseReqFile` method instead? This way the index.js could remain a bit lightweight.